google/cloud/videointelligence/v1/video_intelligence.proto

```diff
@@ -28,6 +28,7 @@ option java_multiple_files = true;
 option java_outer_classname = "VideoIntelligenceServiceProto";
 option java_package = "com.google.cloud.videointelligence.v1";
 option php_namespace = "Google\\Cloud\\VideoIntelligence\\V1";
+option ruby_namespace = "Google::Cloud::VideoIntelligence::V1";
```

Comment:
Is this going to change the existing Ruby proto package name?

Reply:
No. The gem name will stay the same. The generated GAPIC client code is already using VideoIntelligence. This change will resolve an outstanding issue about confusing docs.

https://github.com/googleapis/google-cloud-ruby/issues/1762

Reply:
Great!